app/javascript/react/components/BrandShowTile.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+
+
+const BrandShowTile = (props) => {
+
+    return(
+      <div className='cell small-4 pizzaCard'>
+        <div className='card'>
+          <div className='card-section text-center'  key={props.id}>
+            <Link to={`/pizza/${props.id}`}>{props.name}</Link>
+          </div>
+        </div>
+      </div>
+    )
```

Comment:
tabbed too far in, but this is aligned well. I would use this example for your PizzaShowTile